spack-wrapper

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Spack Root Config
+SPACK=$SNAP_USER_COMMON
+
+# If MODULEPATH not set or null, assign
+MODULEPATH="${MODULEPATH:=$SPACK/share/spack/lmod/$SNAP_ARCH_TRIPLET/Core}"
+
+# Copy over Spack configs
```

Comment:
May want to indicate *why* you are copying over the Spack configs. Context is helpful for any future travelers (yourself included).